docs/actions-cohortextractor.md

```diff
@@ -60,18 +60,14 @@ generate_study_cohort
   outputs:
     highly_sensitive:
       data: output/input.csv.gz
-    ### This file is produced in compressed format. If using Stata for your analysis you may need to change
-    ### the --output-format option to 'csv' and the .gz file extension to produce an uncompressed CSV instead.
 ```
 
-This produces a zipped file to reduce storage space required in the backend. Changing to `--output-format=csv` argument allows an uncompressed CSV to be produced which may be useful while testing in dummy data or if using Stata for analysis of this file. To import the compressed file in Python, you can use `pd.read_csv(<file>, compression='gzip')`. 
+Running the action with `opensafely run generate_study_population` will create the file `output/input.csv.gz`. The file is compressed by default to reduce storage space required and improve performance in the backend.  If you need to view the csv locally for debugging your code, running `gunzip output/input.csv.gz` in a terminal should give you a plain `.csv` to view.
```

Comment:
`gunzip` isn't installed on Windows by default

Reply:
gunzip is installed with git-bash, which currently recommend

Reply:
s/in a terminal/in a git-bash terminal ?

Reply:
or is that now too Windows-specific?

Reply:
`in a terminal (using git-bash on windows)` ?

Reply:
Not the idea that we might have an `opensafely preview` command to do this natively

Reply:
I have added and documented using `opensafely unzip` to view csv's locally